src/Geta.EPi.Extensions/LinkItemExtensions.cs

```diff
@@ -15,26 +15,26 @@ public static class LinkItemExtensions
         ///     Returns ContentReference for provided LinkItem if it is EPiServer content otherwise returns EmptyReference.
         /// </summary>
         /// <param name="source">Source LinkItem for which to return content reference.</param>
-        /// <returns>Returns ContentReference for provided LinkItem if it is EPiServer content otherwise returns EmptyReference.</returns>
+        /// <returns>Returns ContentReference for provided LinkItem if it is EPiServer content otherwise returns EmptyReference. Note that remainingUrl part will be lost use with caution.</returns>
         public static ContentReference ToContentReference(this LinkItem source)
         {
             var content = source.ToContent();
 
             return content != null
-                       ? content.ContentLink
-                       : ContentReference.EmptyReference;
+                ? content.ContentLink
+                : ContentReference.EmptyReference;
         }
 
         /// <summary>
         ///     Returns IContent for provided LinkItem if it is EPiServer content otherwise returns null.
         /// </summary>
         /// <param name="source">Source LinkItem for which to return content.</param>
-        /// <returns>Returns IContent for provided LinkItem if it is EPiServer content otherwise returns null.</returns>
+        /// <returns>Returns IContent for provided LinkItem if it is EPiServer content otherwise returns null. Note that remainingUrl part will be lost use with caution.</returns>
```

Comment:
As it returns content, the comment is wrong. Content doesn't have remaining URL ;)